controllers/metal3datatemplate_controller.go

```diff
@@ -67,98 +67,98 @@ type Metal3DataTemplateReconciler struct {
 // +kubebuilder:rbac:groups="",resources=events,verbs=get;list;watch;create;update;patch
 // +kubebuilder:rbac:groups="",resources=secrets,verbs=get;list;watch;create;update;patch;delete
 
-// Reconcile handles Metal3Machine events.
+// Reconcile handles Metal3DataTemplate events.
 func (r *Metal3DataTemplateReconciler) Reconcile(ctx context.Context, req ctrl.Request) (_ ctrl.Result, rerr error) {
-	metadataLog := r.Log.WithName(dataTemplateControllerName).WithValues("metal3-datatemplate", req.NamespacedName)
+	log := r.Log.WithName(dataTemplateControllerName).WithValues("metal3-datatemplate", req.NamespacedName)
 
 	// Fetch the Metal3DataTemplate instance.
-	capm3DataTemplate := &infrav1.Metal3DataTemplate{}
+	metal3DataTemplate := &infrav1.Metal3DataTemplate{}
 
-	if err := r.Client.Get(ctx, req.NamespacedName, capm3DataTemplate); err != nil {
+	if err := r.Client.Get(ctx, req.NamespacedName, metal3DataTemplate); err != nil {
 		if apierrors.IsNotFound(err) {
 			return ctrl.Result{}, nil
 		}
 		return ctrl.Result{}, err
 	}
-	helper, err := patch.NewHelper(capm3DataTemplate, r.Client)
+	helper, err := patch.NewHelper(metal3DataTemplate, r.Client)
 	if err != nil {
 		return ctrl.Result{}, errors.Wrap(err, "failed to init patch helper")
 	}
-	// Always patch capm3Machine exiting this function so we can persist any Metal3Machine changes.
+	// Always patch the Metal3DataTemplate exiting this function so we can persist any changes.
 	defer func() {
-		err := helper.Patch(ctx, capm3DataTemplate)
+		err := helper.Patch(ctx, metal3DataTemplate)
 		if err != nil {
-			metadataLog.Info("failed to Patch capm3DataTemplate")
+			log.Info("failed to Patch Metal3DataTemplate")
 		}
 	}()
 
 	cluster := &clusterv1.Cluster{}
 	key := client.ObjectKey{
-		Name:      capm3DataTemplate.Spec.ClusterName,
-		Namespace: capm3DataTemplate.Namespace,
+		Name:      metal3DataTemplate.Spec.ClusterName,
+		Namespace: metal3DataTemplate.Namespace,
 	}
 
 	// Fetch the Cluster. Ignore an error if the deletion timestamp is set
 	err = r.Client.Get(ctx, key, cluster)
-	if capm3DataTemplate.ObjectMeta.DeletionTimestamp.IsZero() {
+	if metal3DataTemplate.ObjectMeta.DeletionTimestamp.IsZero() {
 		if err != nil {
-			metadataLog.Info("Error fetching cluster. It might not exist yet, Requeuing")
+			log.Info("Error fetching cluster. It might not exist yet, Requeuing")
 			return ctrl.Result{}, nil
 		}
 	}
 
 	// Create a helper for managing the metadata object.
```

Comment:
may be the comment needs an update as well ?

Reply:
Thanks, fixed!